services/gateway/shared/regex.js

```diff
@@ -36,7 +36,7 @@ const TRANSACTION_EXECUTION_STATUS = /^\b(?:pending|success|fail|,){0,5}\b$/;
 const POS_VALIDATOR_STATUS = /^\b(?:active|standby|banned|punished|ineligible|,){0,9}\b$/;
 const CCM_STATUS = /^\b(?:ok|module_not_supported|module_not_supported|channel_unavailable|recovered|,){0,9}\b$/;
 const DATE_INTERVAL = /^\b(?:(?:\d{4})-(?:(?:1[012])|(?:0?[1-9]))-(?:(?:[012][1-9])|(?:[123]0)|31))(?::(?:(?:\d{4})-(?:(?:1[012])|(?:0?[1-9]))-(?:(?:[012][1-9])|(?:[123]0)|31)))?\b$/;
-const NETWORK_CSV = /^\b(?:mainnet|testnet|betanet|alphanet|devnet|,){0,9}\b$/;
+const NETWORK_CSV = /^\b(?:mainnet|testnet|betanet|devnet|,){0,9}\b$/;
```

Comment:
Since we deleted `alphanet`, we can also update this.

```suggestion
const NETWORK_CSV = /^\b(?:mainnet|testnet|betanet|devnet|,){0,7}\b$/;
```